hwdb.d/60-sensor.hwdb

```diff
@@ -118,6 +118,11 @@ sensor:modalias:acpi:BOSC0200*:dmi:*svnAcer*:*pnSpinSP111-34:*
 
 # Aquarius Cmp NS483
 sensor:modalias:acpi:MXC4005*:dmi:*:svnAquarius*:pnCmp*NS483:*
+ ACCEL_MOUNT_MATRIX=-1, 0, 0; 0, 1, 0; 0, 0, 1
```

Comment:
there's no need to duplicate this one, it's identical to the one below, so keep those merged and just add the new one

Reply:
okay, then I'll remove the comments about the model so as not to cause confusion

Reply:
https://github.com/systemd/systemd/pull/33798

Reply:
just force push to the same branch next time, no need for a different PR